File: frontend/between_pages/lib/models/lists/reading_list_request_dto.dart
/// DTO para solicitar la creación de una nueva lista de lectura.
class ReadingListRequestDTO {
  final String name;
  final String? description;

  ReadingListRequestDTO({
    required this.name,
    this.description,
  });

  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'description': description,
    };
  }
}

File: backend/src/main/java/com/calonuria/backend/features/list/dto/ReadingListRequestDTO.java
package com.calonuria.backend.features.list.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Data;

/**
 * DTO para la creación y edición de listas de lectura.
 */
@Data
public class ReadingListRequestDTO {

    @NotBlank(message = "El nombre de la lista es obligatorio")
    private String name;
    private String description;
}